tests/scr-readingsystem-support/OPS/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-12-10</dc:date>
+    <dc:description>Checks whether the reading system implements the epubReadingSystem object in a spine-level document.</dc:description>
```

Comment:
What does "in a spine-level document" imply here? As opposed to what? I recommend either removing those words or clarifying how this differs from what it *would* mean if those words didn't exist.

I think you mean "as opposed to the iframe test" but I can't imagine an RS implementing ERS in an iframe but *not* in the main document, so I would tend to omit the extra language here, implying that this is the base case, and then have the iframe test as is (essentially, a more specific test).

Reply:
Agreed.